lang/python/pypi.mk

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (C) 2019 Jeffery To
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+ifneq ($(PYPI_NAME),)
```

Comment:
i suspect some usage of `$(strip)` would be recommended here;
i.e.   `ifneq ($(strip $(PYPI_NAME),))`  or something along the lines;

Reply:
Sounds good